storage/store.go

```diff
@@ -657,6 +669,7 @@ func (sc *StoreContext) setDefaults() {
 	}
 
 	raftElectionTimeout := time.Duration(sc.RaftElectionTimeoutTicks) * sc.RaftTickInterval
+	sc.concurrentSnapshotLimit = 5
```

Comment:
I'd like to see this set to 1 until we can measure an improvement from allowing concurrent snapshots.